### ISSUE TITLE
Incorrect build&setup guide at wiki

### ISSUE THREAD

**Issue by Snarkorel** [NETMF/llilum#223]
Setup and build instructions from [wiki](https://github.com/NETMF/llilum/wiki/Setup) says, that I need to install LLVM-3.8.1 binaries and download CFE sources of LLVM 3.8.1.
But BuildLlvmWithVS.cmd script file is configured to the 3.8.0 version of LLVM, same issue for [LlvmApplicaton.props](https://github.com/NETMF/llilum/blob/dev/Zelig/Zelig/CompileTime/Llvm.NET/LibLLVM/LlvmApplication.props).
More than that, I can't build CFE sources due to error:

```
llvm-config not found -- LLVM_CONFIG-NOTFOUND

-- Configuring incomplete, errors occurred!
```

There is llvm-config in LLVM sources, not Clang. So, should I merge Clang and LLVM sources into one directory and try to build, or there is mistake in build instructions at wiki?

UPD: accroding to INSTALL at Clang sources - user should put cfe sources to LLVM_SRCROOT\tools\clang.

**Comment by Snarkorel** [comment]
[Pull request](https://github.com/NETMF/llilum/pull/224) created. Please update the wiki.

**Comment by Snarkorel** [comment]
[Wiki updated](https://github.com/Snarkorel/llilum-wiki), since github doesn't support pull requests for wikis, please add proposed changes.

**Comment by lt72** [comment]
Fixed with 6f2887dd1df77556dacd5f2a7934c9fc84e56acc. 
Thank you!
